packages/typespec-python/test/generated/azure-client-generator-core-internal/_specs_/azure/clientgenerator/core/internal/aio/operations/_operations.py

```diff
@@ -214,7 +214,9 @@ async def public_only(self, *, name: str, **kwargs: Any) -> _models.PublicModel:
         return deserialized  # type: ignore
 
     @distributed_trace_async
-    async def _internal_only(self, *, name: str, **kwargs: Any) -> _models._models.InternalModel:
+    async def _internal_only(  # pylint: disable=protected-access
```

Comment:
Seems the disable could not take affect after auto-format.

Reply:
As long as the disable is after function directly, it works for whole function. For example, the following code won't report "unused-argment" error:
```python
def func( # pylint: disable=unused-argument
        x,
        y
):
    print(x)
```